addons/binding/org.openhab.binding.chromecast.test/src/test/java/org/openhab/binding/chromecast/internal/handler/ChromecastOSGiTest.java

```diff
@@ -1,70 +0,0 @@
-/**
```

Comment:
Do you want to remove this test? If so, please also remove the `itest` directory and its content

Reply:
It's was just a placeholder for adding more tests so I've removed it. The file that got added is the .project file of the itests dir itself. Everytime you import all itests without deselecting the parent it gets automatically created/added.